ckanext/unaids/theme/templates/email/dataset_transfer.txt

```diff
@@ -0,0 +1,12 @@
+Dear {{ user.name }},
+
+    The {{ dataset_org.name }} administrator has requested a transfer of dataset {{ dataset.name }} to your organisation {{ recipient_org.name }}.
```

Comment:
`dataset.name` is `a-slugified-title-of-a-dataset`. You sure you want to use it here? I think you've meant `dataset.title`.